web_src/js/components/ContextPopup.vue

```diff
@@ -120,6 +133,13 @@ export default {
             callback();
           }
         });
+      }).fail(() => {
+        this.loading = false;
+        this.$nextTick(() => {
+          if (callback) {
+            callback();
+          }
+        });
```

Comment:
Some problems here:

1. The code reads strange:
```
$.get(url, (data) => {}).fail( () =>{} )
```

It can be rewritten to either `await` or `$.get().done().fail().always()`

2. Network failure is not the same as "issue doesn't exist".